images/manager/Dockerfile

```diff
@@ -25,15 +25,20 @@ COPY manager /root
 RUN curl -LO https://get.helm.sh/helm-v2.14.1-linux-amd64.tar.gz && \
     tar -zxvf helm-v2.14.1-linux-amd64.tar.gz && \
     chmod +x linux-amd64/helm && \
-    mv linux-amd64/helm /usr/local/bin/helm
-
-RUN git clone https://github.com/nicholasjackson/mtls-go-example && \
-    cd /root/mtls-go-example && \
-    PW=$(pwgen -Bs1 12) && yes | ./generate.sh cortex.example.com $PW && \
-    mkdir ../cortex.example.com && mv 1_root 2_intermediate 3_application 4_client ../cortex.example.com
-
-RUN curl -L https://git.io/getLatestIstio | ISTIO_VERSION=1.2.2 sh - && \
-    mv ./istio-1.2.2/install/kubernetes/helm/istio ./manifests/ && \
-    mv ./istio-1.2.2/install/kubernetes/helm/istio-init ./manifests/
+    mv linux-amd64/helm /usr/local/bin/helm && \
+    rm -rf linux-amd64 && \
+    rm helm-v2.14.1-linux-amd64.tar.gz
+
+RUN PW=$(pwgen -Bs1 12) && \
+    WEBSITE=cortex.example.com && \
+    openssl req \
+       -subj "/C=US/ST=Denial/L=Springfield/O=Dis/CN=$WEBSITE" \
```

Comment:
Do you know why `/C=US/ST=Denial/L=Springfield/O=Dis/`? Is it possible to leave them out, e.g. just do `-subj "/CN=$WEBSITE"`?